eduNEXT/frontend-app-validation-panel
# feat: add studio header and footer

This PR adds the studio header and footer to maintain the look and feed with the legacy code 


![image](https://github.com/eduNEXT/frontend-app-validation-panel/assets/66016493/1c74b1e5-eee1-42e1-93f7-e9c8dc719ee0)

**How to test**:

1. Install this app and branch in dev mode 
```
MFE_COURSE_AUTHORING_PANEL_MFE_APP:
  name: course_authoring
  port: 2001
  repository: https://github.com/eduNEXT/frontend-app-validation-panel.git
  version: dcoa/add-footer-header

```
3. open the app in the browser and see that the header and footer are displayed.
